pulser-core/pulser/backend/remote.py

```diff
@@ -221,6 +221,74 @@ def supports_open_batch(self) -> bool:
         """Flag to confirm this class can support creating an open batch."""
         pass
 
+    @staticmethod
+    def add_measurement_to_sequence(sequence: Sequence) -> Sequence:
```

Comment:
If you make it a public method like this, you should add a Warning in the docstring saying that it will convert all tensors to arrays (which is a very weird side-effect).
I think it would make sense to keep it private, ie
```suggestion
    def _add_measurement_to_sequence(sequence: Sequence) -> Sequence:
```

Reply:
Yeah let's make it private